src/utilities.h

```diff
@@ -630,16 +630,16 @@ std::string makeUniqueId(IdList &idList);
  * Function to support linking units names to their corresponding @c Units items.
  *
  * @param component The component to check.
- * @param issueList A vector of @c IssuePtr items in which unlinked units are recorded for reporting.
+ * @param descriptionList A @c std::vector of @c std::string items in which unlinked units are recorded for reporting.
```

Comment:
This is not what a description list is.

Reply:
The description was "wrong" in the first place, as are some other descriptions (just search for `descriptionList` and you will find one for `traverseComponentEntityTreeLinkingUnits()`). Anyway, happy to change the documentation for this (and the other) `descriptionList` if you tell me what you would like to see.

Reply:
A @c std::vector of variables paired with a description.

Reply:
Done in commit 4090dfc6.